src/main/webapp/scripts/map.js

```diff
@@ -184,14 +127,10 @@ function placeMarkerAndPanTo(latLng, map) {
   });
   /* Recenter the Map */
   map.panTo(latLng);
-  togglePopup();
   /* Update the latitude and longitude values in the popup */
   populateLocation(latLng);
-
-  /* If marker is right-clicked, delete */
-  google.maps.event.addListener(marker, "rightclick", function(event) {
-    marker.setMap(null);
-  });
```

Comment:
because the (grey) marker will now be automatically deleted everytime we close the popup box. Meaning that we won't be able to even right click it (no need to manually remove it).

Reply:
Sounds good. Thanks for clarifying!